.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: false
+dist: xenial
+addons:
+  apt:
+    sources:
+      - sourceline: "ppa:jonathonf/ffmpeg-4"
+    packages:
+      - libudev-dev
+      - libavformat-dev
+      - libavcodec-dev
+      - libavdevice-dev
+      - libavutil-dev
+      - libswscale-dev
+      - libswresample-dev
+      - libavfilter-dev
+matrix:
+  fast_finish: true
+  include:
+    - python: "3.5.3"
+      env: TOXENV=lint
+    - python: "3.5.3"
+      env: TOXENV=pylint
+    - python: "3.5.3"
+      env: TOXENV=typing
+    - python: "3.5.3"
+      env: TOXENV=py35
+    - python: "3.7"
+      env: TOXENV=py37
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+install: pip install -U tox
+language: python
+script: travis_wait 40 tox --develop
+services:
+  - docker
```

Comment:
This can be removed too I think.

Reply:
Cool. I think we are now down to settings that I actually understand :)